WebDevStudios/nextjs-wordpress-starter
# Make example for ACF gutenberg blocks

Closes N/A

### Link

Link to the branch preview generated by Vercel.

### Description

* Duplicate the MediaText block with ACF Gutenberg blocks
  * Now we have an example of the same block using 3 different approaches (Native Gutenberg, Lazy Blocks, and ACF Gutenberg Blocks
  
NOTE: I couldn't figure out how to get the image URL

### Screenshot

![ 2021-02-04 at 16 15 42 ](https://user-images.githubusercontent.com/12365909/106956180-3e702880-6704-11eb-9e7a-476ce0d2956a.png)


### Verification

How will a stakeholder test this?

1. Add an ACF MediaText block to a page (available with https://github.com/WebDevStudios/nextjs-starter-wordpress/pull/7)
2. Verify that it renders
